README.md

```diff
@@ -16,13 +16,11 @@ What is this?
 
 k3s is intended to be a fully compliant Kubernetes distribution with the following changes:
 
-1. Legacy, alpha, non-default features are removed. Hopefully, you shouldn't notice the
-   stuff that has been removed.
-2. Removed most in-tree plugins (cloud providers and storage plugins) which can be replaced
+1. Removed most in-tree plugins (cloud providers and storage plugins) which can be replaced
```

Comment:
I think we should try to more or less mirror what is in the docs at https://rancher.com/docs/k3s/latest/en/